django-stubs/core/validators.pyi

```diff
@@ -7,7 +7,7 @@ from django.core.files.base import File
 EMPTY_VALUES: Any
 
 _Regex = Union[str, Pattern[str]]
-_ValidatorCallable = Callable[[Any], None]
+_ValidatorCallable = Callable[[Any], object]
```

Comment:
I don't think we should relax the types so much. The fact that some type technically works and doesn't crash Django doesn't mean it's the correct type, or that it will continue working with new Django versions.

For example, this makes it valid to pass a validator function that returns boolean True/False. Users might not realize that their validator function doesn't actually do anything, because Django excepts an exception to be raised instead.

Reply:
I agree, `None` is a part of the contract here. You either raise or not.
This can produce new errors for developers, not reduce them.

Reply:
I see, thanks!